Makefile

```diff
@@ -25,10 +25,10 @@ validate: clean
 	tox
 
 test.start_elasticsearch:
-	docker-compose up -d
+	docker compose up -d
```

Comment:
Seems like the minimum version of docker has changed, do we need a change log message?

Reply:
There's no changelog file in this repo but I'll mention it in the release notes when I release the new version.